src/covid_model_seiir_pipeline/io/marshall.py

```diff
@@ -0,0 +1,195 @@
+from contextlib import contextmanager
```

Comment:
I think you wanted `git mv` here since it appears you moved `marshall.py`, updated a few imports, and then added a few lines at the bottom.

But maybe you changed the middle as well?

Reply:
Yeah, sorry.  I was not organized about workflow while doing this.  I pulled off a marshall at a time and used the parallel implementations for testing as I was developing.  

The csv and zip marshalls are essentially the same with some minor tweaks to make them stateless.

I swapped the backend for the hdf marshall to tables (see pr description) and so got rid of lots of the manual handling necessary with h5py.